wave_lang/kernel/wave/water.py

```diff
@@ -185,8 +189,8 @@ def find_binary(name: str) -> str | None:
 
 @lru_cache
 def is_water_available() -> bool:
-    """Returns True if the water_mlir package is available."""
-    return (Path(__file__).parent / "water_mlir" / "water_mlir").exists()
+    """Returns True if the water-opt binary is available."""
```

Comment:
It looks like the check for the package was intentional from here: https://github.com/iree-org/wave/commit/c14db2cdd2633e6377b8027e3fefaca81e2fbc14#diff-e7d61d9650d9dcd96695d2cfb2df154bdc27ab31c432264848386c8808b073fbL183. Are you sure you want to check for the binary instead? Binary is expected to be distributed as part of the package